packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -104,6 +104,11 @@ export class SaveFileDialogProps extends FileDialogProps {
      */
     saveLabel?: string;
 
+    /**
+     * A human-readable value for the input.
+     */
+    inputValue?: string;
```

Comment:
Should not it be used in electron implementation somehow?

Reply:
I need to see why Electron does not display the menu item but the keybinding works and triggers the dialog.
I would like to be able to pass the input value to Electron as well.